cmake/sysbuild/mcuboot.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
```

Comment:
rename file `mcuboot_nrf54h20.cmake` or similar so it's not used by other code/modules